examples/js/loaders/ColladaLoader.js

```diff
@@ -3886,11 +3903,11 @@ THREE.ColladaLoader.prototype = {
 		var textureLoader = new THREE.TextureLoader( this.manager );
 		textureLoader.setPath( this.resourcePath || path ).setCrossOrigin( this.crossOrigin );
 
-		var tgaLoader;
+		var tgaLoader = this.tgaLoader;
 
-		if ( THREE.TGALoader ) {
+		if ( tgaLoader ) {
 
-			tgaLoader = new THREE.TGALoader( this.manager );
+			tgaLoader.manager = this.manager;
```

Comment:
On the off chance the user _had_ created a different manager for the TGALoader, do we want to override it?

Reply:
Oops missed the thread below.